prisma/migrations/20240222032317_drop_territory_notification_settings/migration.sql

```diff
@@ -0,0 +1,15 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `noteTerritoryPosts` on the `users` table. All the data in the column will be lost.
+
+*/
+
+-- migrate values from founders which had notifications for territory posts enabled to new table
+INSERT INTO "SubSubscription"("userId", "subName")
+SELECT u.id, s.name
+FROM users u JOIN "Sub" s ON u.id = s."userId"
+WHERE "noteTerritoryPosts";
+
+-- AlterTable
+ALTER TABLE "users" DROP COLUMN "noteTerritoryPosts";
```

Comment:
We don't want to drop this column on initial deploy as it's a backwards incompatible change with prod and people will get 500's.

Reply:
Fixed in 448f9b5c